### PR TITLE
EZP-26325: Content search instead of Location search

### PR DESCRIPTION
**Pull request by dpobel**
JIRA: https://jira.ez.no/browse/EZP-26325
PR against https://github.com/ezsystems/PlatformUIBundle/pull/712

# Description

**WIP** do a Content search instead of Location search when searching in PlatformUI.

## Tasks

* [x] Add a `findContent` method like the `findLocation` method to search plugin
* [x] Add the `loadLocation` flag to content search
* [x] Replace `findLocation` by `findContent` when searching in PlatformUI.

### CODE REVIEW

**Comment by StephaneDiot** [comment]
ready for review @yannickroger

**Comment by StephaneDiot** [comment]
ping @andrerom : As I won't have any reviewer for the full week, and I know this one is pretty urgent, I wanted to know what should I do with this PR.

**Comment by andrerom** [comment]
Besides question above, we can merge this into your PR, rebase it on latests 1.6 and once passing we can send the issue back to QA.

File: Resources/public/js/views/services/plugins/ez-searchplugin.js [hunk 1]
```diff
@@ -440,6 +536,55 @@ YUI.add('ez-searchplugin', function (Y) {
...
+         * @param {Function} callback
+         */
+        _loadLocationForContent: function (viewName, contentStructArr, callback) {
+            var contentService = this._getContentService(),
+                locationsIds,
+                query;
+
+            locationsIds = Y.Array.reduce(contentStructArr, '', function (previousId, struct, index) {
+                // TODO this is bad but no other way it seems
+                var locationId = struct.content.get('resources').MainLocation.split('/').pop();
```
Comment (andrerom) [inline Resources/public/js/views/services/plugins/ez-searchplugin.js]: note, content might not have location.
So this code and code calling it should have safeguards for that.

Comment (andrerom) [inline Resources/public/js/views/services/plugins/ez-searchplugin.js]: Hmm ignore, this is just theoretical possibility, so not something we need to spend time on now. UI and all other use of the product is currently always creating locations.

**Pull request merged**